pkg/scheduler/plugins/coscheduling/core/gang.go

```diff
@@ -151,7 +152,7 @@ func (gang *Gang) tryInitByPodConfig(pod *v1.Pod, args *config.CoschedulingArgs)
 			gang.Name, pod.Annotations[extension.AnnotationGangGroups])
 	}
 	gang.GangGroup = groupSlice
-
+	gang.GangGroupId = util.GetGangGroupId(groupSlice)
```

Comment:
An optimization can be considered here to ensure that the groupSlice contains at least one member and prevent the group id from being empty.

Reply:
You mean the groupId is a pointer to string?